algorithms/Python/recursion/gcd_using_recursion.py

```diff
@@ -0,0 +1,9 @@
+def gcd(a, b):
+
+	if b==0:
+		return a  
+	else:
+		return gcd(b, a%b)
+
```

Comment:
Just add some basic texts/example calling the gcd function and pr should be good to go 👍